ansible/inventory/sample-k3s/group_vars/all.yml

```diff
@@ -0,0 +1,10 @@
+---
+k3s_version: v1.22.3+k3s1
+ansible_user: debian
+systemd_dir: /etc/systemd/system
+controller_ip: "{{ hostvars[groups['controller'][0]]['ansible_host'] | default(groups['controller'][0]) }}"
+extra_server_args: "--no-deploy traefik"
```

Comment:
It looks like `--disable=traefik` may be the recommended approach now: https://github.com/k3s-io/k3s/issues/1160